src/Bundle/tests/Functional/SwiftmailerSenderTest.php

```diff
@@ -0,0 +1,100 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\MailerBundle\tests\Functional;
+
+use Sylius\Bundle\MailerBundle\tests\Model\SentMessage;
+use Sylius\Bundle\MailerBundle\tests\Provider\MessagesProvider;
+use Sylius\Bundle\MailerBundle\tests\Purger\SentMessagesPurger;
+use Sylius\Component\Mailer\Sender\SenderInterface;
+use Symfony\Bundle\FrameworkBundle\Test\KernelTestCase;
+
+final class SwiftmailerSenderTest extends KernelTestCase
+{
+    private SenderInterface $sender;
+
+    private MessagesProvider $messagesProvider;
+
+    private string $spoolDirectory;
+
+    protected function setUp(): void
+    {
+        self::bootKernel(['environment' => 'test_with_swiftmailer']);
+        $container = self::getContainer();
+
+        $this->sender = $container->get('sylius.email_sender');
+        $this->spoolDirectory = $container->getParameter('kernel.cache_dir') . '/spool/default';
```

Comment:
Where this default come from 🤯  I guess, this is some kind of swiftmailer dark sorcery, but still I'm surprised

Reply:
Indeed, it's added by default by Swiftmailer :dancer: